closed/make/Main.gmk

```diff
@@ -30,7 +30,11 @@ j9vm-build :
 	+($(CD) $(SRC_ROOT)/closed && $(MAKE) -f OpenJ9.gmk SPEC=$(SPEC) VERSION_MAJOR=$(VERSION_MAJOR) build-j9)
 
 j9vm-compose-buildjvm : j9vm-build
```

Comment:
Not sure this is the correct spot for this, I'm just not sure where is atm.  Openjdk should already have a case to handle cross compilation and not be generating a buildjdk in that case.  More than likely we have added the 'j9vm-compose-buildjvm' dependency to the wrong target.

Reply:
make/Main.gmk...
java.base-jmod: j9vm-compose-buildjvm
...should likely be about 10-15 lines down in the CREATE_BUILDJDK ifeq.

Reply:
Yes, Openjdk works entirely as expected (I started with an Openjdk/hotspot build before trying to build OpenJ9). It's certainly something we broke.